app/views/topics/show.html.erb

```diff
@@ -1,31 +1,105 @@
+          ] if @topic.parent.present? ),
+          {
+            field: "Description",
+            value: @topic[:description].present? ? @topic[:description] : "Empty"
+          },
+          {
+            field: "Status",
+            value: govuk_status(@topic)
+          },
+        ],
+        edit: {
+          href: edit_topic_path(@topic),
+          link_text: "Edit page"
+        }
+      } %>
+    </section>
+
+    <% if @topic.can_have_children? %>
+      <section class="govuk-!-margin-bottom-3">
+        <%= render "shared/list-children-pages", page: @topic %>
+      </section>
     <% end %>
 
     <% if @topic.child? %>
-      <% if @topic.published? %>
-        <%= link_to 'Archive', propose_archive_topic_path(@topic) %>
-      <% else %>
-        <%= link_to 'Remove', archive_topic_path(@topic),
-          method: 'post',
-          data: { confirm: 'Are you sure?' } %>
-      <% end %>
+      <section class="govuk-!-margin-bottom-3">
+        <%= render 'shared/list-links-preview', tag: @topic %>
+      </section>
     <% end %>
-  <% end %>
-<% end %>
-
-<%= render 'shared/tags/metadata', resource: @topic %>
-<%= render 'shared/tags/children', resource: @topic %>
-
-<% if @topic.child? %>
-  <%= render 'shared/tags/lists_of_links_preview', tag: @topic %>
-<% end %>
+  </div>
 
-<% if @topic.redirect_routes.any? %>
-  <%= render 'redirects', redirect_routes: @topic.redirect_routes %>
-<% end %>
+  <div class="govuk-grid-column-one-third">
+    <aside class="app-side__wrapper">
+      <div class="app-side">
+        <div class="app-side__actions">
+          <% if @topic.may_publish? %>
+            <div class="govuk-!-margin-bottom-1">
+              <%= link_to "Publish",
+                publish_topic_path(@topic),
+                method: :post,
+                class: %w(govuk-button),
+                data: { module: "govuk-button" } %>
+            </div>
+          <% end %>
+          <% if @topic.can_have_children? %>
+            <div class="govuk-!-margin-bottom-1">
+              <% if gds_editor? %>
+                <%= link_to "Add child page",
+                  new_polymorphic_path(@topic, parent_id: @topic.id),
+                  class: %w(govuk-link govuk-link--no-visited-state) %>
+              <% end %>
+            </div>
+          <% end %>
+          <% if @topic.child? %>
+            <div class="govuk-!-margin-bottom-1">
+              <% if @topic.published? %>
+                <%= link_to 'Archive mainstream browse page',
+                  propose_archive_topic_path(@topic),
+                  class: %w(govuk-link govuk-link--no-visited-state)
+                %>
+              <% else %>
+                <%= link_to 'Delete', archive_topic_path(@topic),
+                  method: 'post',
+                  data: { confirm: 'Are you sure?' },
+                  class: %w(govuk-link app-link--destructive)
+                %>
```

Comment:
```suggestion
                <form action="<%= publish_topic_path(@topic) %>" method="POST">
                  <%= render("govuk_publishing_components/components/button", {
                      text: "Delete",
                      destructive: true
                    }) %>
                </form>
```

Reply:
Just realised this uses a link in collections publisher so i'll do the same here.